BetonQuest-core/src/main/java/pl/betoncraft/betonquest/utils/Updater.java

```diff
@@ -86,8 +86,8 @@ public Updater(File file) {
         }
         String version = plugin.getDescription().getVersion();
         final Version locVer = new Version();
-        isDevBuild = version.contains("dev");
-        isOfficial = isDevBuild && version.contains("#");
+        isDevBuild = version.contains("-SNAPSHOT");
+        isOfficial = isDevBuild && version.contains("#") && !version.contains("#UNOFFICIAL");
```

Comment:
Why do official builds need to contain a #?

Reply:
isOfficial  means a offical Dev-Build.
And the # marks the build number.Since now #UNOFFICIAL mark a self builded jar explicit.
Releases dose not contain a #